lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,37 @@
+require 'httparty'
+
+class EdamamApiWrapper
+  class EdamamError < StandardError; end
+
+  APP_ID = ENV["EDAMAM_APP_ID"]
+  APP_KEY = ENV["EDAMAM_APP_KEY"]
+  BASE_URL = "https://api.edamam.com/search?"
+  BASE_URI = "http://www.edamam.com/ontologies/edamam.owl#recipe_"
+
+  def self.search_results(query)
+    url = "#{BASE_URL}q=#{query}&app_id=#{APP_ID}&app_key=#{APP_KEY}&from=0&to=120"
+
+    response = HTTParty.get(url)
+
+    unless response["count"] > 0
+      raise EdamamError.new
+    end
```

Comment:
You should include an error message here, so that if some other developer (or future you) encounters this error they know what went wrong.